tests/mocks/mock_server_docker.py

```diff
@@ -92,7 +92,9 @@ def assert_server_process_start_called_with(self,
                                                 gpu_uuids,
                                                 http_port=8000,
                                                 grpc_port=8001,
-                                                metrics_port=8002):
+                                                metrics_port=8002,
+                                                mounts=[],
```

Comment:
Using dictionary and lists in the default arguments can lead to strange issues. See here: https://docs.python-guide.org/writing/gotchas/

Reply:
Ah wow I didn't know this. Thanks for pointing this out.